### PR TITLE
Updating panos_tag_object

### PR DESCRIPTION
**Pull request by mrichardson03**
#269

### CODE REVIEW

File: library/panos_tag_object.py [hunk 1]
```diff
@@ -102,166 +87,61 @@
...
+            color=dict(type='str', choices=COLOR_NAMES),
+            comments=dict(type='str'),
+            commit=dict(type='bool', default=True)
+        )
     )
 
-    module = AnsibleModule(argument_spec=argument_spec, supports_check_mode=False)
+    module = AnsibleModule(
+        argument_spec=helper.argument_spec,
+        supports_check_mode=False
```
Comment (shinmog) [inline library/panos_tag_object.py]: This is `True` now.

Also, you're missing `helper.required_one_of`.

File: library/panos_tag_object.py [hunk 2]
```diff
@@ -102,166 +87,61 @@
...
-                existing_obj.color = objects.Tag.color_code(color)
-                existing_obj.comments = comments
-                existing_obj.apply()
-                changed = True
-
-        elif state == 'absent':
-            existing_obj = find_object(device, name, objects.Tag, device_group)
+        listing = Tag.refreshall(parent, add=False)
+    except PanDeviceError as e:
+        module.fail_json('Failed refresh: {0}'.format(e))
```
Comment (shinmog) [inline library/panos_tag_object.py]: `msg='Failed...`

**Comment by shinmog** [comment]
@mrichardson03 
It looks like these comments hold true for the other PRs.

File: library/panos_tag_object.py (hunk 1 above)
Comment (mrichardson03) [reply library/panos_tag_object.py]: This module wasn't using `required_one_of` before.  Should `helper.required_one_of` be specified anyway?

Comment (shinmog) [reply library/panos_tag_object.py]: The default "required_one_of" is making sure they are specifying auth credentials.  Ansible doesn't have support for, "make sure they specify `provider.password` or `provider.api_key`", but it's still doing its best.  So yeah, all modules should at least include `helper.required_one_of`.

**Pull request merged**